### PR TITLE
Replace use of deprecated items

### PR DESCRIPTION
**Pull request by ddgenome**
The HelloWorld example had some outdated ideas, chief among them was
using the Atomist team/workspace ID as the Slack workspace ID.

The NotifyOnPush example was using the deprecated addressChannels
method rather than the send method.

Update tests and mocking as necessary.

Update automation-client to simplify imports.

### CODE REVIEW

File: lib/commands/HelloWorld.ts [hunk 1]
```diff
@@ -49,24 +48,20 @@ export class HelloWorld implements HandleCommand {
     @MappedParameter(MappedParameters.SlackUserName)
     public slackUser: string;
 
-    public handle(ctx: HandlerContext): Promise<HandlerResult> {
+    public async handle(ctx: HandlerContext): Promise<HandlerResult> {
```
Comment (cdupuis) [inline lib/commands/HelloWorld.ts]: I wonder if this is a good pattern to use `async` and then don't use `await` in the function body. WDYT?

Comment (ddgenome) [reply lib/commands/HelloWorld.ts]: Here is why I switched around the type assertions in the tests https://palantir.github.io/tslint/rules/no-object-literal-type-assertion/

Comment (ddgenome) [reply lib/commands/HelloWorld.ts]: Rather, I thought adding the no-object-literal-type-assertion was a good idea for source, so I needed to rearrange the type assertions under test to accommodate that.

**Review by cdupuis — APPROVED** [review]

File: lib/commands/HelloWorld.ts (hunk 1 above)
Comment (cdupuis) [reply lib/commands/HelloWorld.ts]: Makes sense, thanks @ddgenome

**Pull request merged**